src/odemis/gui/cont/tabs/correlation_tab.py

```diff
@@ -245,7 +245,9 @@ def get_display_priority(cls, main_data) -> Optional[int]:
         if main_data.microscope is None:
             # NOTE: microscope = None means the Odemis Viewer
             # Therefore, we want to enable the correlation tab
-            return 1
+            # hide tab by default
+
+            return 0
```

Comment:
`None` means "hidden". 0 is lowest priority. If you want the analysis tab to be higher priority that the correlation tab, you should raise the priority of the analysis tab to 1 (if it's a viewer?).